traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/ParkingItem.java

```diff
@@ -20,7 +20,7 @@
     private URL imageURL;
     private Price price;
     private ProductType productType;
-    private List<ProductItemCategory> categories;
+    private List<BookingItemCategory> categories;
```

Comment:
Why does a ParkingItem pertain to a BookingItemCategory? They don't have the same values.

Reply:
i dont even know why did i change that :/

Reply:
no, we're not parsing categories for any items other than booking. This is because the categories for parking/partner offerings/etc have not been agreed upon in the company